src/__tests__/TransitionParser-test.js

```diff
@@ -24,7 +24,8 @@ describe('TransitionParser', function () {
       background: '#FFF',
       height: '50px',
       width: '50px',
-      transition: 'background 1s 1s, height 2s cubic-bezier(0,0,0,0) 2s, ' +
+      transition: 'background 1s 1s, height 2s ' +
+        'cubic-bezier(0.25, 0, .45, 2) 2s, ' +
```

Comment:
Would you mind to add multiple cubic-bezier to this transition? You could just change the linear function following this line